assets/css/main.css

```diff
@@ -170,6 +170,28 @@ body {
   text-decoration: none;
 }
 
+#Explanation{
+  font-size: 12pt; 
+}
+
+#Toggle-Explanation{
+  background: #f1652991;
+  font-size: 12pt;
+  border: none;
+  color: white ;
+  padding: 12px 25px;
+  text-align: center;
+  margin: 2px 1px;
+  opacity: 0.6;
```

Comment:
`opacity` can cause issues with accessibility because it lowers the contrast of the element (especially since this is a button with text).

Can this be removed?